src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -28,6 +29,35 @@ internal static class Constants
     /// </summary>
     public class ModuleIntrinsics
     {
+#if !UNIX
+        /// <summary>
+        /// Compares a PATH entry on Windows. Treats / and \ as the same and
+        /// compares using OrdinalIgnoreCase.
+        /// </summary>
+        private class WindowsPathEnvComparer : IEqualityComparer<char>
```

Comment:
Can it be `struct` or `readonly struct`?

Reply:
I’m not sure, can it? Do we gain anything by using a struct here?

Reply:
Done